src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -18,6 +18,8 @@ export class RadioButton extends Widget {
 	private inputElement: HTMLInputElement;
 	private _onClicked = new Emitter<void>();
 	public readonly onClicked: Event<void> = this._onClicked.event;
+	private _onChanged = new Emitter<boolean>();
```

Comment:
I started to question the purpose of onClicked event, do we still need that?

Reply:
I can remove that but it might break existing functionality.

Reply:
onClick implies an event that's fired whenever the element is actually clicked by the user - where as onChanged would be whenever the value is changed.

The primary difference I could see is that onClicked would only be fired for the one that was actually clicked - but onChanged would be fired for both it and the previously selected one that had its value changed.

Reply:
So while I think there's a clear difference it's not clear to me when we'd actually care about the onClick scenario I gave. I'm kind of fine either way but I think having an onChanged event as well is clearer as to when it's actually fired.